devito/function.py

```diff
@@ -1074,3 +1090,108 @@ def __indices_setup__(cls, **kwargs):
     @classmethod
     def __shape_setup__(cls, **kwargs):
         return kwargs.get('shape', (kwargs.get('nt'), kwargs.get('npoint'),))
+
+
+class PrecomputedSparseFunction(AbstractSparseFunction):
+    is_PrecomputedSparseFunction = True
```

Comment:
we need ``is_PrecomputedSparseFunction = False`` in ``class Basic``, within ``types.py``

Reply:
Done